pages/_document.tsx

```diff
@@ -0,0 +1,25 @@
+import type { DocumentContext, DocumentInitialProps } from 'next/document';
+import Document, { Html, Head, Main, NextScript } from 'next/document';
+
+class MyDocument extends Document {
+  public static async getInitialProps<T extends DocumentContext>(
```

Comment:
No need for getInitialProps in the document file, I will approve the PR .. just remove the getInitialProps and merge